src/transformers/commands/user.py

```diff
@@ -173,6 +207,85 @@ def run(self):
         print("Done")
 
 
+class ListReposObjsCommand(BaseUserCommand):
+    def run(self):
+        token = HfFolder.get_token()
+        if token is None:
+            print("Not logged in")
+            exit(1)
+        try:
+            objs = self._api.list_repos_objs(token, organization=self.args.organization)
+        except HTTPError as e:
+            print(e)
+            print(ANSI.red(e.response.text))
+            exit(1)
+        if len(objs) == 0:
+            print("No shared file yet")
+            exit()
+        rows = [[obj.filename, obj.lastModified, obj.commit, obj.size] for obj in objs]
+        print(tabulate(rows, headers=["Filename", "LastModified", "Commit-Sha", "Size"]))
+
+
+class RepoCreateCommand(BaseUserCommand):
+    def run(self):
+        token = HfFolder.get_token()
+        if token is None:
+            print("Not logged in")
+            exit(1)
+        try:
+            stdout = subprocess.run(["git", "--version"], capture_output=True).stdout.decode("utf-8")
+            print(ANSI.gray(stdout.strip()))
+        except FileNotFoundError:
+            print("Looks like you do not have git installed, please install.")
+
+        try:
+            stdout = subprocess.run(["git-lfs", "--version"], capture_output=True).stdout.decode("utf-8")
+            print(ANSI.gray(stdout.strip()))
+        except FileNotFoundError:
+            print(
+                ANSI.red(
+                    "Looks like you do not have git-lfs installed, please install."
+                    " You can install from https://git-lfs.github.com/."
+                    " Then run `git lfs install` (you only have to do this once)."
+                )
+            )
+        print("")
+
+        user, _ = self._api.whoami(token)
+        namespace = self.args.organization if self.args.organization is not None else user
+
+        print("You are about to create {}".format(ANSI.bold(namespace + "/" + self.args.name)))
+
+        if not self.args.yes:
+            choice = input("Proceed? [Y/n] ").lower()
+            if not (choice == "" or choice == "y" or choice == "yes"):
+                print("Abort")
+                exit()
+        try:
+            url = self._api.create_repo(token, name=self.args.name, organization=self.args.organization)
+        except HTTPError as e:
+            print(e)
+            print(ANSI.red(e.response.text))
+            exit(1)
+        print("\nYour repo now lives at:")
+        print("  {}".format(ANSI.bold(url)))
+        print("\nYou can clone it locally with the command below," " and commit/push as usual.")
+        print(f"\n  git clone {url}")
```

Comment:
Do you have any specific cases when you use f-strings and when you use .format? You seem to use both throughout. I am curious because if this is a style-guide thing that you want in the library, then perhaps it can be added to CONTRIBUTING.md so that people know when to use f-strings and when to use format.

Reply:
Ooops no I really use them interchangeably. 

BTW I cc'ed you on this as you updated the implem of the file_utils's networking calls in the past if I remember correctly.

Reply:
f-strings are supposed to be faster than format nowadays (not that it matters for this).

Reply:
@julien-c You have a good memory! Yes, it was a user request where they wanted to make offline usage easily possible.